lib/config/presets/internal/workarounds.ts

```diff
@@ -88,4 +89,14 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  ignoreDotnet7Preview: {
+    description: 'Ignore dotnet 7 preview releases',
+    packageRules: [
+      {
+        matchDatasources: ['docker'],
+        matchPackagePrefixes: ['mcr.microsoft.com/dotnet/'],
+        allowedVersions: '!/^7\\.0/',
```

Comment:
```suggestion
        matchCurrentVersion: '<7',
        allowedVersions: '<7',
```

Reply:
You should update the `description` in your suggestion, to something like:

`Require dashboard approval for dotnet 7 or higher`.